File: ÖBB-Prüfstand/ÖBB-Prüfstand.Core/Entities/OEBBFunktion.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OEBB_Pruefstand.Core.Entities
{
    public class OEBBFunktion 
    {
        
        public string Beschreibung { get; set; }
        public int Headersize { get; set; }
        public int Entries { get; set; }
        public int EchtEntries { get; set; }
        public int NumRead { get; set; }
        public List<OEBBMessergebnisse> Messergebnisse { get; set; }
    }
}

File: asuqz_frontend/src/models/oebb-funktion.ts
import { OEBBMessergebnisse } from './oebb-messergebnisse';
export class OEBBFunktion{
  constructor(
    public id: number,
    public beschreibung: string,
    public headersize: number,
    public entries: number,
    public echtEntries: number,
    public numRead: number,
    public messergebnisse: OEBBMessergebnisse[]
  ){}
}
